src/components/layout/AppBar.vue

```diff
@@ -93,7 +93,30 @@
           <span>{{ $t(`app.general.label.turn_device_${topNavPowerDeviceOn ? 'off' : 'on'}`, { device: topNavPowerToggle }) }}</span>
         </v-tooltip>
       </div>
-
+      <div
+        v-if="authenticated && socketConnected && topNavPinLight"
+      >
+        <v-tooltip bottom>
+          <template #activator="{ on, attrs }">
+            <app-btn
+              fab
+              small
+              :elevation="0"
+              class="mr-1 bg-transparent"
+              color="transparent"
+              :disabled="!klippyReady"
```

Comment:
I know we currently don't do it for the top nav power toggle, but maybe the button should be disabled when there's a wait on `onSetOutputPin` to indicate a pending state change?

Reply:
```suggestion
              :disabled="!klippyReady || hasWait($waits.onSetOutputPin)"
```
Good idea. I tried "hasWait" and I don't see disabling of the button because it's too fast, but I think it works.